VVResonances/interactive/makeCard.py

```diff
@@ -4,9 +4,9 @@
 from CMGTools.VVResonances.statistics.DataCardMaker import DataCardMaker
 cmd='combineCards.py '
 
-purities=['HPHP','HPLP','LPLP']
 purities=['HPHP','HPLP']
 signals = ["BulkGWW"]
+
 for sig in signals:
   for p in purities:
 
```

Comment:
I know its not in this pull request, but I would still like to introduce a change here:
card=DataCardMaker('',p,'13TeV',35900,'JJ_%s'%sig) -> cat='_'.join(['JJ',sig,p,'13TeV'])
    card=DataCardMaker('',p,'13TeV',35900,'JJ',cat)
instead of introducing the name of the signal in the tag of the datacard maker i would suggest to leave that out and instead add the signal to the name of the datacard seperately by chaning the following lines in DataCardMaker:
line 8: def __init__(self,finalstate,category,period,luminosity=1.0,physics="LJ",cat=""):
line 17: self.cat = cat; if self.cat!="": self.rootFile = ROOT.TFile("datacardInputs_"+cat+".root","RECREATE")
        else: self.rootFile = ROOT.TFile("datacardInputs_"+self.tag+".root","RECREATE")
and line 1176: if self.cat!="":f = open("datacard_"+self.cat+'.txt','w')
        else: f = open("datacard_"+self.tag+'.txt','w')

The reason for this is that otherwise we have to change all the names in runFitPlots.py (and other scripts that use the RooWorkspace) because they will contain the tag: JJ_BulkWW_13TeV instead of JJ_13TeV

Reply:
Go ahead!